pyproject.toml

```diff
@@ -150,22 +150,8 @@ docstring-code-format = true
 testpaths = ["tests"]
 filterwarnings = [
   "error",
-  'ignore:distutils Version classes are deprecated:DeprecationWarning',
-  'ignore:In the future `np.bool`',
-  'ignore:make_block is deprecated and will be removed',
-  'ignore:np.find_common_type is deprecated',
-  'ignore:is_sparse is deprecated and will be removed',
-  'ignore:Passing a BlockManager to DataFrame is deprecated',
   'ignore:.*defaulting to pandas implementation',
   'ignore:.*implementation has mismatches with pandas',
-  'ignore:.*Do not use the `random` module inside strategies',
-  'ignore:.*You are using pyarrow version',
```

Comment:
we can probably keep this one turned off globally, as it just warns about pyarrow==14 being insecure so it's not going to create issues for us going forwards (unlike, say, silencing a deprecation warning)

Reply:
Thanks